cnf-certification-test/accesscontrol/rbac/automount.go

```diff
@@ -45,3 +48,45 @@ func (at *AutomountToken) AutomountServiceAccountSetOnSA(serviceAccountName, pod
 	}
 	return sa.AutomountServiceAccountToken, nil
 }
+
```

Comment:
Rename argument "ch" argument to something else, to distinguish from struct "ch" in:
func NewAutomountTokenTester(ch *clientsholder.ClientsHolder) *AutomountToken {
	return &AutomountToken{
		ch: ch,
	}
}

Reply:
Sounds good to me.  I'll clarify that one.

Reply:
Fixed.